app/views/admin/world_locations/show.html.erb

```diff
@@ -7,7 +7,7 @@
       <span class="name"><%= @world_location.name %></span>
       &ldquo;<span class="title"><%= @world_location.title %></span>&rdquo;
     </h1>
-    <p><%= view_on_website_link_for @world_location %></p>
```

Comment:
Should `view_on_website_link_for` behave differently when passed a world_location?